bitnami/mongodb-sharded/values-production.yaml

```diff
@@ -704,6 +704,11 @@ service:
   ## Extra ports to expose (normally used with the `sidecar` value)
   ##
   extraPorts:
+  ## Specify the sessionAffinity setting for the service. Can be "None" or "ClientIP".
+  ## If "ClientIP", consecutive client requests will be directed to the same mongos Pod
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+  ##
+  sessionAffinity:
```

Comment:
Why did you put this different in `values.yaml` and `values-production.yaml`?

Reply:
I looked at the value above in values-production.yaml and my head somehow came to the conclusion that it should be like that :D

Can change it tomorrow, but semantically it doesn't really matter since there is the default value in the template ...